crates/nu-command/src/viewers/table.rs

```diff
@@ -60,12 +65,17 @@ impl Command for Table {
             .input_output_types(vec![(Type::Any, Type::Any)])
             // TODO: make this more precise: what turns into string and what into raw stream
             .named(
-                "start-number",
-                SyntaxShape::Int,
-                "row number to start viewing from",
-                Some('n'),
+                "theme",
+                SyntaxShape::String,
+                "set a table mode/theme",
+                Some('t'),
+            )
+            .named(
+                "index",
+                SyntaxShape::Any,
+                "set or set off a index mode/theme",
+                Some('i'),
             )
-            .switch("list", "list available table modes/themes", Some('l'))
```

Comment:
I'm not sure why --list is removed here? We still need it to show people what all the different modes/themes are.

Reply:
It'll be shown if a unknown theme is picked

Reply:
```nu
~/bin/nushell> ls | table --theme basic222342
Error: nu::shell::cant_convert

  × Can't convert to theme.
   ╭─[source:1:1]
 1 │ nu
   · ▲
   · ╰── can't convert string to theme
   ╰────
  help: expected either 'basic', 'thin', 'light', 'compact',
        'with_love', 'compact_double', 'rounded', 'reinforced',
        'heavy', 'none', 'psql', 'markdown', 'dots', 'restructured',
        'ascii_rounded', or 'basic_compact'
```

Reply:
I think it's good to have an error like this when there's a typo, but if you're looking for the complete list of what table modes nushell supports, you should be able to ask for it. If you have to create an error, just to see the list, it's not very discoverable or easy to figure out.

Reply:
restored

Reply:
Thank you!